Modify S2078(multiple languages): Update to the education framework (APPSEC-115) (#1209)
SonarSource/rspec: rules/S2078/impact.adoc

=== What is the possible impact?

> In the context of a web application vulnerable to LDAP injection: +
> After discovering the injection point, attackers insert data into the

Reviewer: ```suggestion
In the context of a web application vulnerable to LDAP injection, after discovering the injection point, attackers insert data into the
```

Author: Ah, I'm not aligned with this suggestion, because it makes the paragraph bigger, and an even-bigger sentence

Author: ```suggestion
In the context of a web application vulnerable to LDAP injection:
after discovering the injection point, attackers insert data into the
```

Author: Agreed on this last syntactically correct fix

vulnerable field to execute malicious LDAP commands.

The impact of this vulnerability depends on how vital LDAP servers are to the
organization.

Below are some real-world scenarios that illustrate the various impacts of an
attacker exploiting the vulnerability.

==== Data leakage or corruption

In typical scenarios where systems perform innocuous LDAP operations to find
users or create inventories, an LDAP injection could result in data 
leakage or corruption.

==== Privilege escalation

> A malicious LDAP query could allow an attacker to impersonate an average user
> or administrator in scenarios where systems perform authorization checks or

Reviewer: ```suggestion
A malicious LDAP query could allow an attacker to impersonate a low-privileged user
or an administrator in scenarios where systems perform authorization checks or
```
